api/project.go

```diff
@@ -228,21 +221,22 @@ func ValidateRepositorySchemaPathTemplate(schemaPathTemplate string, tenantMode
 	for _, token := range tokens {
 		tokenMap[token] = true
 	}
+
+	allowedTokens := schemaPathTemplateTokens
 	if tenantMode == TenantModeTenant {
-		if _, ok := tokenMap[EnvironmentToken]; ok {
-			return &common.Error{Code: common.Invalid, Err: errors.Errorf("%q is not allowed in the template for projects in tenant mode", EnvironmentToken)}
-		}
+		allowedTokens = tenantSchemaPathTemplateTokens
```

Comment:
So we don’t support tenant mode projects set the `schemaPathTemplate`? The `tenantSchemaPathTemplateTokens` is empty and likes like will return error in line 239

Reply:
We still support schemaPathTemplate but with a fixed path without any tokens, most likely .LATEST.sql alone.